plugins/inputs/statsd/statsd.go

```diff
@@ -348,7 +348,7 @@ func (s *Statsd) parseStatsdLine(line string) error {
 				return errors.New("Error Parsing statsd line")
 			}
 			m.floatvalue = v
-		case "c", "s":
+		case "c", "s", "m", "mr":
```

Comment:
Are you sure these will never be floats? What are they anyways, do they act like counters or like gauges?